winforms/src/toga_winforms/widgets/datepicker.py

```diff
@@ -12,7 +12,12 @@ def create(self):
         self.native = WinForms.DateTimePicker()
 
     def get_value(self):
-        return datetime.datetime.strptime(self.native.Text, "%A, %B %d, %Y").date()
+        return datetime.datetime.strptime(
+            self.native.Value.ToString(
+                "yyyy-MM-ddTHH:mm:sszzz", CultureInfo.InvariantCulture
+            ),
+            "%Y-%m-%dT%H:%M:%S%z",
```

Comment:
*eyeballs twitch at the inconsistent capitalisation of time formatting characters...*

🤪

Reply:
Yes!